AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/MiscUtils.java

```diff
@@ -32,35 +32,6 @@
 
```

Comment:
@larsgrefer, I can't since @SimonMarquis did not create a branch for this. I'll merge this after the necessary changes have been done and will now push our release without these changes.